README.md

```diff
@@ -6,6 +6,25 @@ If you are looking for the ArcGIS Maps SDK for JavaScript core package, it is av
 
 This repository is organized as a monorepo using yarn workspaces.
 
+## Available Templates
+There are download links in the readme for each of the following templates. We recommend downloading a template from the links below and using your package manager of choice.  
+
+- Coding Components
+    - [AMD Script Tag](./packages/coding-components/templates/amd-script-tag)
+    - [Angular](./packages/coding-components/templates/angular)
+    - [React](./packages/coding-components/templates/react)
+    - [Vite](./packages/coding-components/templates/vite)
+    - [Vue](./packages/coding-components/templates/vue)
+    - [Webpack](./packages/coding-components/templates/webpack)
```

Comment:
This README may eventually become too populated once other component packages are added. These lists of templates could be wrapped in a `<details>` tag to save space? 

Example:

Map components
- AMD...
- Angular...
- etc...

Coding components
<details> <summary>Click to expand</summary>

- AMD...
- Angular...
- etc...
</details>

WDYT?

Reply:
maybe we do it when that day comes? Right now it feels like it's hiding content that doesn't need to be hidden.